medical_pathology/views/medical_pathology_view.xml

```diff
@@ -0,0 +1,114 @@
+                    <filter string="Affected Chromosome"
+                            domain="[]"
+                            context="{'group_by':'chromosome'}"
+                            />
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record id="medical_pathology_view_tree" model="ir.ui.view">
+        <field name="name">medical.pathology.tree</field>
+        <field name="model">medical.pathology</field>
+        <field name="arch" type="xml">
+            <tree string="Medical Pathology">
+                <field name="name" />
+                <field name="code" />
+                <field name="code_type_id" />
+                <field name="category_id" />
+                <field name="protein" />
+                <field name="gene" />
+                <field name="chromosome" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="medical_pathology_view_form" model="ir.ui.view">
+        <field name="name">medical.pathology.form</field>
+        <field name="model">medical.pathology</field>
+        <field name="arch" type="xml">
+            <form string="Medical Pathology">
+                <header />
+                <sheet>
+                    <div class="oe_title" name="title">
+                        <label for="name" class="edit_only"/>
+                        <h1><field name="name"/></h1>
+                    </div>
+                    <div class="oe_right oe_button_box" name="buttons" />
+                    <group name="data">
+                        <group name="main">
+                            <field name="code" />
+                            <field name="code_type_id" />
+                            <field name="category_id" />
+                        </group>
+                        <group name="genetics">
+                            <field name="protein" />
+                            <field name="chromosome" />
+                            <field name="gene" />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page string="Disease Groups" name="notes" autofocus="autofocus">
+                            <field name="medical_pathology_group_ids" />
+                        </page>
+                        <page string="Others" name="others">
+                            <separator string="Notes"/>
+                            <field name="notes" />
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="medical_pathology_action" model="ir.actions.act_window">
+        <field name="name">Pathologies</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">medical.pathology</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_id">tree,form</field>
```

Comment:
@bmw95,

Were you going for view_id for this field or is this a duplicate of view_mode?